.ubeu TLD

```diff
@@ -0,0 +1,66 @@
+
+A UBeU digital identity token is in the standardized format ‘firstname.lastname.ubeu.’ and is created by tokenizing the subdomain of a surname SLD. By establishing a standardized format for digital identity tokens, the UBeU platform can leverage the hierarchal nature of the KNS protocol to allow ‘.ubeu’ digital identity token holders to create an unlimited number of custom secondary subdomains (‘*.firstname.lastname.ubeu’) which allows users to name resources and map tokenized digital assets to designated wallets. This capability extends the utility of the UBeU platform beyond digital identity to also being a digital asset management platform and an effective tool for risk mitigation, as digital assets can be efficiently organized and partitioned among wallets.
+
+Through the development of open-source software, tools, and SDKs, the UBeU unified digital identity registry will be decentralized so that wallets and DApps throughout the Hedera ecosystem can interact directly with the registry. By decentralizing access to the registry, wallets will be able to integrate digital identity creation and the issuance of a digital identity NFT into the account creation process. A digital identity NFT authorizes users to access and interact with the UBeU DApp to configure secondary subdomains. As well the decentralization of the UBeU platform will enable third-party Hedera DApps to interact directly with the UBeU registry. This has several advantages, including the ability to resolve names in a DApp, the ability for a DApp to support reverse resolution, and it allows users to configure secondary subdomains to point at a DApps resources, and provides an easy way for users to update a name they own to point at a DApp's resources. It is anticipated that as a unified digital registry with standardized naming formats, the UBeU platform will enable 3rd party services to be built on top of it. The benefit of using less complex account identifiers in favor of custom account aliases and the need for a digital asset management platform to accommodate the tokenization of real-world consumer assets justifies the `.ubeu` TLD.
+
+## Clearance
+
+| Organization       | Cleared? |
+| ------------------ | -------- |
+| IANA               | TRUE     |
+| Hashgraph.name     | TRUE     |
+| KNS                | TRUE     |
+| UnstoppableDomains | TRUE     |
+
+## Specification
+A top-level domain (TLD) '.ubeu' smart contract maintains the registry of domains, records the owner, the resolver, and allows the owner to make changes to the data therein. A registrar smart contract is written owning the secondary-level domains and issues subdomain NFTs of '.ubeu' domains to buyer. Domain resolution is made using KNS resolver. The subdomain NFT allows its owner to interact with the UBeU app and create secondary subdomains.
+
+The UBEU TLD should be issued as `.ubeu`.
+
+This TLD will be issued as a KNS TLD NFT.
+
+The admin of this TLD shall be 0.0.3724158.
+
```

Comment:
The '.ubeu' TLD will be administered as a private TLD, giving the admin sole authority to register domains under the '.ubeu' TLD.